vue.config.js

```diff
@@ -45,6 +45,20 @@ module.exports = {
     }
   },
   chainWebpack: config => {
+    config.module
+      .rule("fonts")
+      .test(/\.(ttf|otf|eot|woff|woff2)$/)
+      .use("url-loader")
+      .loader("url-loader")
+      .tap(options => {
+        options = {
+          // limit: 10000,
+          name: "./fonts/[name].[ext]",
+          publicPath: process.env.NODE_ENV === "production" ? "./" : "/"
+        };
+        return options;
+      })
+      .end();
```

Comment:
As we are working towards heimdall server 2 we need to test this change to make sure it doesn't break anything there.